tests/integration/example_test_env_vars.yaml

```diff
@@ -11,3 +11,15 @@ post:
   ref: e1be527f39ec31323f0454f7d1422c6260b00580
   # Package managers
   pkg_managers: ["gomod"]
+# Test data for the Valid data in request test
+get:
+  repo: https://github.com/release-engineering/retrodep.git
+  ref: e1be527f39ec31323f0454f7d1422c6260b00580
+  pkg_managers: ["gomod"]
+  # The name of the package
+  packages_names: "github.com/release-engineering/retrodep/v2"
+  # The names of fetched dependencies
+  dependencies_names: ["gopkg.in/yaml.v2", "gopkg.in/check.v1", "golang.org/x/tools", "golang.org/x/text",
```

Comment:
Consider using the syntax:
```
dependencies_names:
- gopkg.in/yaml.v2
- gopkg.in/check.v1
...
```
Order them alphabetically, and it should much easier to find items in it :)